src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -28,7 +28,8 @@
     'azure-cli-core',
     'azure-mgmt-web==0.31.0',
     # v1.17 breaks on wildcard cert https://github.com/shazow/urllib3/issues/981
-    'urllib3[secure]==1.16'
+    'urllib3[secure]==1.16',
+    'pyOpenSSL>=0.16.2',
```

Comment:
I prefer 16.2.0 as it fixes this bug [#568](https://github.com/pyca/pyopenssl/issues/568)